pkg/uefi/file.go

```diff
@@ -178,23 +178,24 @@ func (a *fileAttr) setLarge(large bool) {
 }
 
 // Checks if we need to checksum the file body
```

Comment:
HasChecksum checks ...

Reply:
I'll fix it, but it's true the linter has no problem, even with:

golint -min_confidence=0.0 file.go